packages/ExternalLink/src/ExternalLink.styles.js

```diff
@@ -1,17 +1,30 @@
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import tokens from "@paprika/tokens";
 import { truncateText } from "@paprika/stylers/lib/includes";
 import { toInt } from "@paprika/stylers/lib/helpers";
 
+const contentHoverFocusStyles = css`
+  text-decoration: none;
+
+  &:hover,
+  &:focus {
+    text-decoration: underline;
+  }
+`;
+
 export const ExternalLink = styled.a`
   align-items: center;
   border-radius: ${tokens.border.radius};
   color: ${tokens.textColor.link};
   display: inline-flex;
   max-width: 100%;
   position: relative;
+  text-decoration: none;
```

Comment:
Maybe it was only Storybook, but this is needed to actually remove the underline when `hasNoUnderline={true}`.